docs/stable/serve/storage_aware_scheduling.md

```diff
@@ -2,135 +2,67 @@
 sidebar_position: 0
 ---
 
-# Storage Aware Scheduling
+# Storage Aware Scheduling with Docker Compose
 
 ## Pre-requisites
-To enable storage aware model loading scheduling, a hardware configuration file is required.
-For example, the following is a sample configuration file for two servers:
-```bash
-echo '{
-  "0": {
-    "host_size": "32GB",
-    "host_bandwidth": "24GB/s",
-    "disk_size": "128GB",
-    "disk_bandwidth": "5GB/s",
-    "network_bandwidth": "10Gbps"
-  },
-  "1": {
-    "host_size": "32GB",
-    "host_bandwidth": "24GB/s",
-    "disk_size": "128GB",
-    "disk_bandwidth": "5GB/s",
-    "network_bandwidth": "10Gbps"
-  }
-}' > hardware_config.json
-```
 
-We will use Docker to run a ServerlessLLM cluster in this example. Therefore, please make sure you have read the [Docker Quickstart Guide](../getting_started/docker_quickstart.md) before proceeding.
+We will use Docker Compose to run a ServerlessLLM cluster in this example. Therefore, please make sure you have read the [Docker Quickstart Guide](../getting_started/docker_quickstart.md) before proceeding.
 
 ## Usage
-Start a local Docker-based ray cluster.
 
-### Step 1: Start Ray Head Node and Worker Nodes
+Start a local Docker-based ray cluster using Docker Compose.
 
-1. Start the Ray head node.
+### Step 1: Clone the ServerlessLLM Repository
+
+If you haven't already, clone the ServerlessLLM repository:
 
 ```bash
-docker run -d --name ray_head \
-  --runtime nvidia \
-  --network sllm \
-  -p 6379:6379 \
-  -p 8343:8343 \
-  --gpus '"device=none"' \
-  serverlessllm/sllm-serve
+git clone https://github.com/ServerlessLLM/ServerlessLLM.git
+cd serverlessllm/examples/storage_aware_scheduling
```

Comment:
There is an inconsistency in the path naming: please standardize to "ServerlessLLM" for consistency across the project.

Reply:
Modified. I just follow the pattern in Docker Quickstart Guide, so I guess another pr is needed to modify that.

Reply:
Modified. I just follow the pattern in Docker Quickstart Guide, so I open another pr #154 to modify that.